src/sbd-inquisitor.c

```diff
@@ -966,9 +966,9 @@ int main(int argc, char **argv, char **envp)
         }
 
         value = getenv("SBD_SYNC_RESOURCE_STARTUP");
-        if(value) {
-            sync_resource_startup = crm_is_true(value);
-        }
+        sync_resource_startup =
+            crm_is_true(value?value:SBD_SYNC_RESOURCE_STARTUP_DEFAULT);
```

Comment:
So far without `SBD_SYNC_RESOURCE_STARTUP` being explicitly set in sysconfig, a `SBD_SYNC_RESOURCE_STARTUP_DEFAULT=yes` doesn't actually take effect, since pacemaker side requires an explicit setting anyway, right?

Reply:
Yes. It would even be problematic as graceful-shutdown-detection of pacemaker by sbd might not work properly. On pacemaker-side I've used a patch so far. Proper switch is coming.